### PR TITLE
Initial port

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Review by blackbaud-conorwright — DISMISSED** [review]
Looking good :+1:

File: package.json [hunk 1]
```diff
@@ -19,13 +19,28 @@
...
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@skyux/animations": ">=3.0.0-rc.0",
+    "@skyux/forms": ">=3.0.0-rc.0",
+    "@skyux/grids": "3.0.0-alpha.1",
+    "@skyux/indicators": ">=3.0.0-rc.1",
+    "@skyux/layout": ">=3.0.0-rc.0",
+    "@skyux/list-builder": "3.0.0-alpha.0",
+    "@skyux/lists": ">=3.0.0-rc.0",
+    "@skyux/lookup": ">=3.0.0-rc.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: I know I've said differently in the past, but peer dependencies should only use the caret `^` for now, to ensure that only pre-release versions are accepted.

File: package.json [hunk 2]
```diff
@@ -19,13 +19,28 @@
...
+    "@skyux/modals": ">=3.0.0-rc.0",
+    "@skyux/popovers": ">=3.0.0-rc.0",
+    "microedge-rxstate": "^2.0.2",
+    "rxjs": "^5.4.3"
```
Comment (Blackbaud-SteveBrush) [inline package.json]: Remove `rxjs`

File: package.json [hunk 3]
```diff
@@ -19,13 +19,28 @@
...
+    "@skyux/animations": ">=3.0.0-rc.0",
+    "@skyux/forms": ">=3.0.0-rc.0",
+    "@skyux/grids": "3.0.0-alpha.1",
+    "@skyux/indicators": ">=3.0.0-rc.1",
+    "@skyux/layout": ">=3.0.0-rc.0",
+    "@skyux/list-builder": "3.0.0-alpha.0",
+    "@skyux/lists": ">=3.0.0-rc.0",
+    "@skyux/lookup": ">=3.0.0-rc.0",
+    "@skyux/modals": ">=3.0.0-rc.0",
+    "@skyux/popovers": ">=3.0.0-rc.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: I only saw imports from the following SKY UX modules. Should the other ones be removed?
```
"@skyux/grids": "^3.0.0-alpha.1",
"@skyux/indicators": "^3.0.0-rc.1",
"@skyux/list-builder": "^3.0.0-alpha.0"
```

File: package.json [hunk 4]
```diff
@@ -19,13 +19,28 @@
...
-    "url": "https://github.com/blackbaud/skyux-sample/issues"
+    "url": "https://github.com/blackbaud/skyux-list-builder-view-grids/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-list-builder-view-grids#readme",
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@skyux/animations": ">=3.0.0-rc.0",
+    "@skyux/forms": ">=3.0.0-rc.0",
+    "@skyux/grids": "3.0.0-alpha.1",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: Some of these peer dependencies need carets `^`.

**Review by Blackbaud-SteveBrush — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: done.

File: package.json (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: done.

File: package.json (hunk 4 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: got em.

File: package.json (hunk 3 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: Without them, we have unmet peer dependencies from list-builder, and the app won't serve properly. I can always install them as dev-dependencies to get the local serve to work, but leave them out of peer dependencies and rely on the consumer to make sure they're installed. Whats the right move here?

Comment (Blackbaud-SteveBrush) [reply package.json]: I would put the missing modules (the ones your library does not directly import) in the dev dependencies.

Comment (Blackbaud-AlexKingman) [reply package.json]: done.

**Review by Blackbaud-SteveBrush — DISMISSED** [review]

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**